ql/src/test/queries/clientpositive/describe_table.q

```diff
@@ -1,4 +1,9 @@
 --! qt:dataset:srcpart
+set hive.describe.partitionedtable.ignore.stats=true;
+describe formatted `srcpart`;
```

Comment:
Not sure if partitions have stats. The output is same with and without the config set.

Reply:
Updated please see new comment.